gradle-baseline-java/src/main/groovy/com/palantir/baseline/plugins/BaselineErrorProne.java

```diff
@@ -195,6 +232,76 @@ public void apply(Project project) {
         });
     }
 
+    private Action<ErrorProneOptions> configureErrorProne(
+            Project project,
+            JavaCompile javaCompile,
+            JavaVersion jdkVersion,
+            BaselineErrorProneExtension errorProneExtension,
+            Provider<File> refasterRulesFile,
+            Task compileRefaster) {
+        return errorProneOptions -> {
+            errorProneOptions.setEnabled(true);
+            errorProneOptions.setDisableWarningsInGeneratedCode(true);
+            errorProneOptions.check("EqualsHashCode", CheckSeverity.ERROR);
+            errorProneOptions.check("EqualsIncompatibleType", CheckSeverity.ERROR);
+            errorProneOptions.check("StreamResourceLeak", CheckSeverity.ERROR);
+            errorProneOptions.check("InputStreamSlowMultibyteRead", CheckSeverity.ERROR);
+            errorProneOptions.check("JavaDurationGetSecondsGetNano", CheckSeverity.ERROR);
+            errorProneOptions.check("URLEqualsHashCode", CheckSeverity.ERROR);
+
+            if (isStrict(project) && !isRefactoring(project)) {
+                Set<String> compilerArgs = ImmutableSet.of("-Werror", "-Xlint:deprecation", "-Xlint:unchecked");
```

Comment:
thoughts on enabling these in strict mode?

Reply:
We already bootstrap new projects with `"-Werror", "-Xlint:deprecation"` so I am all for it